### PR TITLE
[MPS] Fixes for MPS testConsistency

### PR DESCRIPTION
**Pull request by kulinseth**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by razarmehr — APPROVED** [review]
Looks good.

File: test/test_mps.py [hunk 1]
```diff
@@ -3332,8 +3332,14 @@ def helper(shape_x, shape_y, shape_z):
             self.assertEqual(cat, cat_cpu)
 
         helper([2, 2, 4, 5], [2, 3, 4, 5], [2, 5, 4, 5])
-        # Empty test - Currently failing! Empty tensor not handled!
-        # helper([0, 2, 4, 5], [2, 0, 4, 5], [2, 5, 0, 5])
+        helper([2, 2, 6, 5], [2, 3, 6, 5], [2, 5, 6, 5])
```
Comment (albanD) [inline test/test_mps.py:3335]: Can this be tested via OpInfo below?

**Review by albanD — APPROVED** [review]
SGTM

File: test/test_mps.py (hunk 1 above)
Comment (kulinseth) [reply test/test_mps.py:3335]: Right, So this specific test without empty tensors can be tested with OpInfo. But this test-case in-general isolated few issues which were missed by TestConsistency, that's why I left it in.

**Pull request closed without merging**